### PR TITLE
Add slack notifications

### PR DESCRIPTION
**Pull request by ikstream**
They are very Basic by now only containing the information about the type of event. Additional Information will be added.

The notification currently looks litke:
Voltage dip < 10%
Voltage dip >= 10%
Voltage above 110%

### CODE REVIEW

File: powqutyd/Makefile [hunk 1]
```diff
@@ -25,14 +25,21 @@ config POWQUTYD_NO_MQTT
...
 endmenu
 endef
 
 
 define Package/powqutyd
 	SECTION:=utils
 	CATEGORY:=Utilities
-	DEPENDS:=+libconfig +kmod-usb-acm +!POWQUTYD_NO_MQTT:libmosquitto +libuci
+	DEPENDS:=+libconfig +kmod-usb-acm \
+		 +!POWQUTYD_NO_MQTT:libmosquitto +libuci \
```
Comment (thuehn) [inline powqutyd/Makefile]: put libuci in  a separate line for better readability

File: powqutyd/files/src/uci_config.c [hunk 2]
```diff
@@ -67,7 +67,7 @@ int uci_config_powquty(struct powquty_conf* conf) {
 			str = uci_lookup_option_string(uci, s, "dev_uuid");
 			if (str == NULL)
 				continue;
-			if (strlen(str) >= MAX_STR_LEN) {
+			if (strlen(str) > MAX_LENGTH) {
```
Comment (thuehn) [inline powqutyd/files/src/uci_config.c]: should'nt this one be a: if (strlen(str) >= MAX_LENGTH - 1)

File: powqutyd/files/include/event_handling.h [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/*
+ * event_handling.h
+ *
+ *  Created on: Jun 27, 2017
+ *  	Author: ikstream
+ */
+#ifndef _EVENT_HANDLING_H_
+
```
Comment (ikstream) [inline powqutyd/files/include/event_handling.h]: done

File: powqutyd/Makefile (hunk 1 above)
Comment (ikstream) [inline powqutyd/Makefile]: done

File: powqutyd/files/src/uci_config.c (hunk 2 above)
Comment (ikstream) [inline powqutyd/files/src/uci_config.c]: True, must somehow missed this. Is fixed now

**Comment by thuehn** [comment]
As I do see merging conflicts, I think the problem is with commit "	Add event handling basics" 22296a8 ... this one got already merged in the former pull request or ?

**Comment by ikstream** [comment]
I don't know what happened during the merge. The web interface marked all issues as solved, but an additional edit for the merge conflict in calculation.h was needed.

One part of the merge conflict was the first commit, but the second part where changes introduced due to the input file pull request

**Comment by thuehn** [comment]
looks good ... thx & merged

**Pull request merged**